app/static/src/app/types/responseTypes.ts

```diff
@@ -177,3 +177,11 @@ export interface SessionMetadata {
     label: string | null,
     friendlyId: string | null,
 }
+
+export interface Versions {
```

Comment:
yes, this will change soon as we get the discrete time/stochastic models here - see the schema https://github.com/mrc-ide/odin.api/blob/main/inst/schema/root.json which defines it as being a loosely defined set